app/src/sharedTest/java/org/oppia/android/app/resumelesson/ResumeLessonFragmentTest.kt

```diff
@@ -295,6 +299,62 @@ class ResumeLessonFragmentTest {
     )
   }
 
+  @Test @Config(qualifiers = "w360dp-h640dp-xxhdpi")
```

Comment:
Robolectric defines it's own device dimensions if none is provided, and those are consistent for both local and CI runs.